Week-7/Assignment7.1/contiguousarray.js

```diff
@@ -0,0 +1,15 @@
+function conSubArray(A) {
+  let maxsum = 0;
```

Comment:
Variable naming should be CamelCase in JS. first letter of variable small and remaining first letter Capital, remaining all small.